openapi.yaml

```diff
@@ -1067,6 +1067,9 @@ components:
         compression_algorithm:
           description: Compression algorithm for the snapshot archive
           type: string
+        cardano_node_version:
+          description: Version of the Cardano node which is used to get information (stake, immutables etc.)
```

Comment:
```suggestion
          description: Version of the Cardano node which is used to create the snapshot archive
```

Reply:
Mithril uses the Cardano node only to gather information in order to create artifacts. The Cardano node does not create artifacts on its own.

Reply:
Same as above :slightly_smiling_face: